.github/workflows/lint-tests-release.yml

```diff
@@ -95,6 +95,19 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version: 1.18
+      - name: Set up arm64 cross compiler
+        run: sudo apt-get -y install gcc-aarch64-linux-gnu
+      - name: Checkout osxcross
+        uses: actions/checkout@v2
+        with:
+          repository: tpoechtrager/osxcross
+          path: osxcross
+      - name: Build osxcross
+        run: |
+          cd osxcross
+          wget https://github.com/phracker/MacOSX-SDKs/releases/download/11.3/MacOSX11.3.sdk.tar.xz -O tarballs/MacOSX11.3.sdk.tar.xz
```

Comment:
should we verify a checksum on this file?

Reply:
not a bad idea. adding